ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2

```diff
@@ -9,6 +9,9 @@ management api gnmi
    transport grpc {{ vrf }}
 {%             if management_api_gnmi.enable_vrfs[vrf].access_group is arista.avd.defined %}
       ip access-group {{ management_api_gnmi.enable_vrfs[vrf].access_group }}
+{%             endif %}
+{%             if management_api_gnmi.set_send_time is arista.avd.defined %}
```

Comment:
Has to defined **true**. Also the data-model is not in compliance with the cli_config. Please use the knob ```notification_timestamp_send_time: < true | false >```

Reply:
fair point, done